wolox-books/src/views/Register.vue

```diff
@@ -76,24 +78,20 @@ export default {
     }
   },
   methods: {
-    signIn () {
+    submit () {
```

Comment:
This should be an async function.

e.g.:
    async loginUser() {
      if (this.$v.$invalid) {
        this.showErrors = true
      } else {
        const { email, password } = this
        if (email && password) {
          this.showErrors = false
          await this.login({ email, password })
        }
      }
    }